contracts/GSN/bouncers/GSNBouncerSignature.sol

```diff
@@ -46,7 +47,11 @@ contract GSNBouncerSignature is Initializable, GSNBouncerBase {
             address(this) // Prevents replays in multiple recipients
         );
         if (keccak256(blob).toEthSignedMessageHash().recover(approvalData) == _getTrustedSigner()) {
+             if (_getTrustedSigner() != address(0)){
```

Comment:
We generally assume throughout `contracts-ethereum-package` that everything has been properly initialized. Note that if something has been properly initialized it is guaranteed not to have the zero address as trusted signer.

Given the consequences of not properly initializing this contract in particular, we might want to add this additional check here, but IIRC that would be a departure from what we've done with all the other contracts in the library.

We generally prefer to keep code as "vanilla" as possible here, to simplify merging back the new releases from upstream. (Otherwise the diffs become very difficult to manage.)

@nventuro What do you think we should do here?

Reply:
I would remove this check both for consistency with the rest of the lib and the need for both repos to be as similar as possible. Many (all?) contracts have very dire consequences if they are not properly initialized.

Reply:
If we agree to remove this check we can close this PR and follow the usual "merge from upstream" procedure to bring over the changes from https://github.com/OpenZeppelin/openzeppelin-contracts/pull/1920.